src/spaceone/cost_analysis/model/data_source_model.py

```diff
@@ -83,14 +90,12 @@ class DataSource(MongoModel):
             "data_source_id",
             "workspace_id",
             "name",
-            "state",
             "data_source_type",
             "secret_type",
             "provider",
```

Comment:
The `schedule` field is necessary here.

```suggestion
            "provider",
            "schedule"
```